GameData/Kerbalism/System/Resources.cfg

```diff
@@ -5,7 +5,7 @@
 RESOURCE_DEFINITION
 {
   name = Atmosphere
-  density = 0.001251          // 1 m^3 of nitrogen at STP
+  density = 0.000001251          // nitrogen at STP in units metric tonne/L or Mg/dm^3 (factor 1 million smaller than typical units kg/m^3)
```

Comment:
I don't think changing the density is a good idea for the resources because all densities used in Kerbalism are kg/m^3